stratum/hal/lib/barefoot/bf_pipeline_utils_test.cc

```diff
@@ -15,8 +15,8 @@ namespace stratum {
 namespace hal {
 namespace barefoot {
 
-const auto& bf_config_1pipe_str =
-   R"pb(device: 1
+static constexpr auto& bf_config_1pipe_str =
```

Comment:
Not sure if the use of `auto` here is actually a benefit. `char*` is not particularly long to write and might be helpful to C++ novices (What's the type of a string literal?). The style guide agrees: 

> do not use it merely to avoid the inconvenience of writing an explicit type

https://google.github.io/styleguide/cppguide.html#Type_deduction

Reply:
good point. I was mostly being lazy. I'll update the type :)